package.json

```diff
@@ -15,12 +15,16 @@
     "discord.js": "^12.5.1",
     "electron": "^11.1.1",
     "express": "^4.17.1",
+    "fetch": "^1.1.0",
```

Comment:
I would prefer to stick to either axios or fetch if we are going to be using one or the other. Can you switch your code from using fetch to using axios?  Here's some differences between the two: https://stackoverflow.com/questions/40844297/what-is-difference-between-axios-and-fetch

Reply:
Sounds good!